From 8f5c9bea431551b4ea407e40b30b59ac7eb46507 Mon Sep 17 00:00:00 2001
From: Blance Tang <guofeng.tang@amlogic.com>
Date: Thu, 24 Jun 2021 21:09:24 +0800
Subject: [PATCH] RDK: fix compile rtl8822CS error [1/1]

PD#SWPL-51742

Problem:
  not exist path $(KERNEL_SRC)/$(M)

  Solution:
    disable check the folder

RDK: Fix

Signed-off-by: Blance Tang <guofeng.tang@amlogic.com>
Change-Id: Ia30c46642a1137afde40bf95f9ef373bf2cf6e19
---
 rtl88x2CS/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/rtl88x2CS/Makefile b/rtl88x2CS/Makefile
index 3dbc0f5..59babd8 100644
--- a/rtl88x2CS/Makefile
+++ b/rtl88x2CS/Makefile
@@ -2266,7 +2266,7 @@ CROSS_COMPILE ?= /4.4_S905L_8822bs_compile/gcc-linaro-aarch64-linux-gnu-4.9-2014
 ifndef KSRC
 EXTRA_CFLAGS += -w
 KSRC := $(KERNEL_SRC)
-EXTRA_CFLAGS += $(foreach d,$(shell find $(shell cd $(KERNEL_SRC)/$(M);pwd) -type d),$(shell echo " -I$(d)"))
+#EXTRA_CFLAGS += $(foreach d,$(shell find $(shell cd $(KERNEL_SRC)/$(M);pwd) -type d),$(shell echo " -I$(d)"))
 # To locate output files in a separate directory.
 #KSRC += O=/4.4_S905L_8822bs_compile/KERNEL_OBJ
 endif
-- 
2.29.2

